modules/optimization/test/tests/optimizationreporter/general_opt/point_loads/adjoint.i

```diff
@@ -0,0 +1,124 @@
+[Mesh]
```

Comment:
This test is going to be the new version that will need to be documented on this page
https://mooseframework.inl.gov/modules/optimization/examples/forceInv_pointLoads.html

so it should have something like this at the top of this file:
```
# DO NOT CHANGE THIS TEST
# this test is documented as an example in forceInv_pointLoads.md
# if this test is changed, the figures will need to be updated.
```

Reply:
I am not changing the documentation for optimization reporter yet. When I do that I will make sure to update the tops of the file where we change tests and not just add new ones.